tests/test_util.py

```diff
@@ -13,8 +13,11 @@
 @pytest.mark.parametrize('test_input, expected', [
```

Comment:
tests are missing the case where the port in the netloc string is different from the default.